crates/router/src/connector/worldline/transformers.rs

```diff
@@ -13,8 +13,9 @@ use crate::{
     types::{self, api, storage::enums},
 };
 
-static CARD_REJEX: Lazy<HashMap<CardProduct, Result<Regex, regex::Error>>> = Lazy::new(|| {
+static CARD_REGEX: Lazy<HashMap<CardProduct, Result<Regex, regex::Error>>> = Lazy::new(|| {
     let mut map = HashMap::new();
+    // Reffered regex pattern from https://gist.github.com/michaelkeevildown/9096cd3aac9029c4e6e05588448a8841
```

Comment:
```suggestion
    // Reference: https://gist.github.com/michaelkeevildown/9096cd3aac9029c4e6e05588448a8841
```

There's a typo: ~Reffered~ Referred